config/v1/types_infrastructure.go

```diff
@@ -1188,6 +1208,19 @@ type VSpherePlatformFailureDomainSpec struct {
 	// +kubebuilder:validation:Required
 	Zone string `json:"zone"`
 
+	// regionAffinity holds the type of region
+	// +openshift:validation:featureGate=VSphereHostVMGroupZonal
+	// +kubebuilder:validation:Optional
```

Comment:
Conversely to required, we prefer the plain version here
```suggestion
	// +optional
```